README.md

```
@@ -394,23 +396,24 @@ plugins {
 
 #### 2. Configure the plugin
 
+Full gradle plugin API specification (see specific examples [here](#gradle-plugin-samples)):
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)